pkg/policy/policy.go

```diff
@@ -59,7 +59,7 @@ var InputTypeForTarget = map[Target]string{
 }
 
 var allTargets = []Target{
-	Terraform, TerraformPlan, Cloudformation, Kubernetes, Helm, Docker, Secrets,
+	Terraform, TerraformPlan, Cloudformation, Kubernetes, Helm, Docker, Secrets, ARM,
```

Comment:
Do we have still have secrets and docker scanning?

Reply:
I believe so, the commands are still available for them as far as I can tell.
I've only added ARM to this list as the `policy vet` command was ignoring ARM policies otherwise.